styles/main.css

```diff
@@ -321,11 +323,11 @@ div.gallery img{
 }
 
 div.gallery a.row-span-2{
-    grid-row: 2;
+    grid-row: span 2;
```

Comment:
Using just a number makes the image start on that line in the grid.  YOu need to include the `span` keyword to have it span that many rows/cols